botathon/index.php

```diff
@@ -662,11 +649,82 @@ function switchActive(e,i){
                 $('html, body').animate({
                     scrollTop: $(hash).offset().top
                 }, 800, function () {
-
                     // Add hash (#) to URL when done scrolling (default click behavior)
                     window.location.hash = hash;
                 });
             } // End if
         });
+        let scrollTop = windowSelector.scrollTop()
+        let scrollBottom = scrollTop + windowSelector.height()
+        for (let i = 0; i < 7; i++) {
```

Comment:
This is still only specifically looping through 7 items, it should use the length of the selectors array I think

Reply:
Changed in 6909ab5